portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -63,6 +63,11 @@
 	<td>//p[contains(text(),'${key_fieldLabel}')] | //label[contains(text(),'${key_fieldLabel}')] | //legend[contains(text(),'${key_fieldLabel}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>LABEL_BY_ROW</td>
+	<td>//div[contains(@class,'ddm-field-container')]//div[@class='ddm-field'][${key_rowNumber}]//p[contains(text(),'${key_fieldLabel}')] | //div[contains(@class,'ddm-field-container')]//div[@class='ddm-field'][${key_rowNumber}]//label[contains(text(),'${key_fieldLabel}')] | //div[contains(@class,'ddm-field-container')]//div[@class='ddm-field'][${key_rowNumber}]//legend[contains(text(),'${key_fieldLabel}')]</td>
```

Comment:
@lima-rafa, why do we need 3 variations of this path (items split by `|`)?

Reply:
Renato for the same reason he needs the path named LABEL, I just gave "power" to xpath to be selected regardless of the line it is. You can look at LABEL's xpath, hear only a punctual addition.

<td>LABEL</td>
<td>//p[contains(text(),'${key_fieldLabel}')] | //label[contains(text(),'${key_fieldLabel}')] | //legend[contains(text(),'${key_fieldLabel}')]</td>

Reply:
I see, @lima-rafa, but we need these 3 variantions for this specific test?

Reply:
@lima-rafa, please keep this path with just what is demanded by this test